packages/frontend-2/components/workspace/JoinPage.vue

```diff
@@ -64,6 +64,8 @@
         </FormButton>
         <FormButton
           v-if="!hasApprovedWorkspace"
+          v-tippy="!canClickCreate ? cantClickCreateReason : undefined"
```

Comment:
@andrewwallacespeckle @gjedlicska In a realistic usecase atm, will new users always see other workspace in the list? If their list is empty and this button disabled it might be a bad experience